charts/bulk-scan-processor/values.preview.template.yaml

```diff
@@ -36,6 +36,7 @@ java:
     PROCESS_PAYMENTS_ENABLED: "false"
     OCR_VALIDATION_URL_BULKSCAN_SAMPLE_APP: "http://bulk-scan-sample-app-aat.service.core-compute-aat.internal"
     SCAN_DELAY: "2000"
+    LEASE_ACQUIRE_DELAY: "120"
```

Comment:
worth naming env to indicate if it is in seconds ? `LEASE_ACQUIRE_DELAY_IN_SECS`

Reply:
updated